angularjs-portal-frame/src/main/webapp/portal/main/partials/welcome-modal-template.html

```diff
@@ -0,0 +1,12 @@
+<div class="welcome-popup center">
+  <h1 ng-bind-html="welcome.title"></h1>
+  <img src="{{ welcome.img }}" class="hidden-xs">
+	<div class="sr-only" tabindex="0" id="welcomeContent" aria-labelledby="welcomeContent">
```

Comment:
I'm not sure how the accessibility for this works, I'll do some research and talk to @jiayinjx. The only reason I didn't put the description as alt text on the image is that it was a really long description, and it seemed too long for alt text. But I'm starting to think it would be better as alt text.